tests/search_disk_index.cpp

```diff
@@ -51,8 +51,8 @@ int search_disk_index(diskann::Metric &metric, const std::string &index_path_pre
                       const std::string &result_output_prefix, const std::string &query_file, std::string &gt_file,
                       const unsigned num_threads, const unsigned recall_at, const unsigned beamwidth,
                       const unsigned num_nodes_to_cache, const _u32 search_io_limit, const std::vector<unsigned> &Lvec,
-                      const float fail_if_recall_below, const bool use_reorder_data = false,
-                      const std::string &filter_label = "")
+                      const float fail_if_recall_below, const std::vector<std::string> &query_filters,
+                      const bool use_reorder_data = false)
```

Comment:
Is this backward compatiable when there are no filters?

Reply:
Yes, the code is indeed backward compatible. The decision point is when query_filters, is empty.